x-pack/legacy/plugins/apm/public/selectors/chartSelectors.ts

```diff
@@ -61,7 +58,6 @@ export function getTransactionCharts(
   });
 
   return {
-    noHits,
     tpmSeries,
     responseTimeSeries
   };
```

Comment:
I think this file can also be moved to the backend (https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/apm/server/lib/transactions/charts/get_timeseries_data/transform.ts) but that's probably out of scope for this PR :)

Reply:
Agree on both counts. Would be easier if all endpoints spit out TimeSeries, but also feel like this PR is getting a bit big for a bug fix.